tests/incremental/hygiene/load_cached_hygiene.rs

```diff
@@ -7,7 +7,7 @@
 //    This causes hygiene information to be saved to the incr cache.
 // 2. One function is the foreign crate is modified. This causes the
 //    optimized mir for an unmodified function to be loaded from the
-//@    incremental cache and written out to the crate metadata.
+//    incremental cache and written out to the crate metadata.
```

Comment:
I'm kinda curios if this was a bug in the migration or if the previous logic actually treated this as a directive, lol.